docs/data/elasticsearch/error.json

```diff
@@ -193,7 +192,7 @@
                     },
                     "filename": "file/name.py",
                     "function": "foo",
-                    "library_frame": true,
+                    "library_frame": false,
```

Comment:
any specific reason to flip all of these?  Concerned that since it's the default we're losing coverage on the `true` code path.

Reply:
There is still one `library_frame: true` left.
The reason for flipping is to get more variety, which we need for the next feature. 
Usually I'd add it to the next PR, but since I had all of this already in I left this.